hitl/logging.cpp

```diff
@@ -0,0 +1,34 @@
+#include <cstdio>
+#include <cstdarg>
+#include "error/error.h"
+#include "absl/flags/flag.h"
+#include "absl/synchronization/mutex.h"
+
+ABSL_FLAG(bool, verbose, false, "Enable verbose logging.");
+static ABSL_CONST_INIT absl::Mutex logging_lock(absl::kConstInit);
```

Comment:
why not use `std::mutex`?

Reply:
https://abseil.io/about/design/mutex

That being said it's mostly habit and we could use std mutex if you prefer.

Reply:
No, let's keep abseil